main/ui/src/main/resources/fxml/unlock.fxml

```diff
@@ -48,15 +48,17 @@
 			<VBox spacing="6" HBox.hgrow="ALWAYS">
 				<FormattedLabel format="%unlock.passwordPrompt" arg1="${controller.vaultName}" wrapText="true"/>
 				<NiceSecurePasswordField fx:id="passwordField" disable="${controller.userInteractionDisabled}"/>
-				<CheckBox fx:id="savePasswordCheckbox" text="%unlock.savePassword" onAction="#didClickSavePasswordCheckbox" disable="${controller.userInteractionDisabled}" visible="${controller.keychainAccessAvailable}"/>
+				<CheckBox fx:id="savePasswordCheckbox" text="%unlock.savePassword" onAction="#didClickSavePasswordCheckbox" disable="${controller.userInteractionDisabled}"
+						  visible="${controller.keychainAccessAvailable}"/>
```

Comment:
Good example, where a hard wrap is annoying and doesn't improve readability. Can we disable the limit for xml files?

Reply:
I disabled the hard line wraps.

Resolved by c376b060d48a07159353e6e7f35af55227d96fb0.